portal/models/questionnaire.py

```diff
@@ -0,0 +1,47 @@
+"""Questionnaire module"""
+from sqlalchemy.dialects.postgresql import JSONB
+
+from ..database import db
+
+
+class Questionnaire(db.Model):
+    __tablename__ = 'questionnaires'
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.Text, nullable=False, unique=True)
```

Comment:
I chose the name 'title' to match the [FHIR spec](https://www.hl7.org/fhir/questionnaire.html).  The db field is unique.

Within the spec, you'll note we have the standard identifiers that can be added.  I see 'name' is also there.  (i'm taking this discussion to skype...)